.github/pull_request_template.md

```diff
@@ -1,11 +1,3 @@
----
-name: Pull Request Standard
-about: You want to include modifications to the code base
-title: ''
```

Comment:
Maybe just keep `title`?

Reply:
This is a bit redundant with the already existing title of the PR :slightly_smiling_face: 

Also it looks like this part that of the template which is between the `---` delimiters is used only in the case of multiple PR templates in order to show a select menu (as in the `.github/ISSUE_TEMPLATE` dir and the [new issue page](https://github.com/input-output-hk/mithril/issues/new/choose))